cargo.json

```diff
@@ -1,7 +1,8 @@
 {
     "groups": {
-        "all": ["crates"]
+        "all": ["@alexcrichton", "@huonw", "@wycats"]
```

Comment:
Because I think cargo should be independent of the crates published on crates.io.